Makefile

```diff
@@ -46,6 +46,8 @@ export CONFIG_VIDEO_OV02C10 = m
 export CONFIG_VIDEO_OV2740 = m
 export CONFIG_VIDEO_HM2170 = m
 export CONFIG_VIDEO_HI556 = m
+export CONFIG_VIDEO_OV13858_INTEL = m
```

Comment:
For"Build with kernel source tree" case, we need to add a Kconfig option for VIDEO_OV13858_INTEL, then in .config set it as "m" and set original VIDEO_OV13858 to "n" or  "not set". Please follow other sensors part and update the README.md.